Cargo.toml

```diff
@@ -125,7 +127,7 @@ substrate-wasm-builder = { git = "https://github.com/paritytech/polkadot-sdk", t
 cumulus-primitives-aura = { git = "https://github.com/paritytech/polkadot-sdk", tag = "polkadot-stable2412-2", default-features = false }
 cumulus-primitives-storage-weight-reclaim = { git = "https://github.com/paritytech/polkadot-sdk", tag = "polkadot-stable2412-2", default-features = false }
 cumulus-client-cli = { git = "https://github.com/paritytech/polkadot-sdk", tag = "polkadot-stable2412-2" }
-cumulus-client-consensus-aura = { git = "https://github.com/paritytech/polkadot-sdk", tag = "polkadot-stable2412-2" }
+cumulus-client-consensus-aura = { git = "https://github.com/paritytech/polkadot-sdk", tag = "polkadot-stable2412-2", features = ["full-pov-size"] }
```

Comment:
We already had problems in the past with this during a migration. Wrong benchmarks yielded a higher PoV size than measured, and when using 100% of the PoV size there is no room for these eventualities.

Reply:
I used this to allow Mykola to quickly reproduce and run benchmarks on the test network, to get it into production we will modify this `full-pov-size` to use 80% of the max, so that we cover what you mentioned.

Reply:
@Moliholy I think this is the issue that you have been hitting: https://github.com/paritytech/polkadot-sdk/issues/6020#issuecomment-2744330650, right ?

Reply:
Moved this part in https://github.com/paritytech/project-mythical/pull/293.